.github/workflows/agw-build-publish-container.yml

```diff
@@ -188,3 +188,69 @@ jobs:
       registry: ${{ needs.build-containers.outputs.registry }}
       test_targets: extended_tests
     secrets: inherit
+
+  publish-container-tests:
+    runs-on: ubuntu-20.04
+    if: always() && github.event_name == 'push'
```

Comment:
Why is `github.event_name == 'workflow_dispatch'` not included here?

Reply:
The results should only be published to the dashboard in the case of a merged PR (as with the publishing steps of e.g. LTE integ tests where you have the same condition for publishing).